lib/API/secrets.json

```diff
@@ -0,0 +1,3 @@
+{
+    "apikey": "onlywehavethiskey"
+}
```

Comment:
blunder pushing this please add to gitignore. thank the lord that we can change headers. also be sure to call this a api_token but key